safepay-typescript-client/src/apis/OrderApi.ts

```
@@ -0,0 +1,244 @@
+    /**
+     * <p>Perform the <code>GENERATE_CAPTURE_CONTEXT</code> action. Generates a JWT token that is used to generate the card fields on the checkout form. The capture context essentially establishes a payment session.</p> <p>The capture context is necessary for generating the Cybersource Flex Microform. These are card fields hosted on Cybersource that are used to capture the customer\'s card details. The form is embedded in your checkout UI.</p> <p>For more details on how to generate the form, please refer to the <a href=\"https://developer.cybersource.com/docs/cybs/en-us/digital-accept-flex/developer/all/rest/digital-accept-flex/microform-integ-v2.html\">Microform </a> guide. Specifically, you should refer to the <a href=\"https://developer.cybersource.com/docs/cybs/en-us/digital-accept-flex/developer/all/rest/digital-accept-flex/microform-integ-v2/micro-getting-started-pay-card/micro-getting-started-pay-card-cs-setup.html\">Client Side</a> documentation. This process should help you obtain the <strong>transient token</strong> JWT from the card fields which you will require for the next action.</p> <p>The capture context also specifies a target origin for which it is valid. By default, this is set to Safepay Checkout. However, this may be modified by authenticating as a merchant via the Secret Key and passing the <code>origin</code> as follows:</p> <pre class=\"click-to-expand-wrapper is-snippet-wrapper\"><code class=\"language-json\">{   \"payload\": {     \"origin\": \"https://api-example.com\"   } }  </code></pre> <p>If targeting Safepay Checkout, this can be left as an empty body: <code>{}</code>. Note that setting a custom origin is only honoured when authenticated as a merchant. For merchant authentication, refer to the <em>API Authentication</em> section.</p> 
+     * Generate Capture Context
+     */
+    async postGenerateCaptureContextRaw(requestParameters: PostGenerateCaptureContextRequest, initOverrides?: RequestInit | runtime.InitOverrideFunction): Promise<runtime.ApiResponse<object>> {
+        const queryParameters: any = {};
+
+        const headerParameters: runtime.HTTPHeaders = {};
+
+        headerParameters['Content-Type'] = 'application/json';
+
+        if (this.configuration && this.configuration.accessToken) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)